src/idl_gen_cpp.cpp

```diff
@@ -494,8 +495,8 @@ class CppGenerator : public BaseGenerator {
 
   // Return a C++ type from the table in idl.h
   std::string GenTypeBasic(const Type &type, bool user_facing_type) const {
-    static const char * const ctypename[] = {
-    // clang-format off
+    static const char *const ctypename[] = {
+// clang-format off
```

Comment:
Not sure why it is doing this, but this change is not desirable. It may need a blank line somewhere to work, I seem to recall.

Reply:
Yeah, clang-format seems to be confused about how to properly align the comments themselves while inside a block. Just putting the on/off comments outside the block makes everything work consistently, fixed in 117e3b0679209f2aa55cbee18c4036e7da4bd4b3.